database/ioc/src/Makefile

```diff
@@ -1,4 +1,4 @@
-TOP=../..
+TOP=../../..
```

Comment:
On 08/16/2016 06:52 PM, Andrew Johnson wrote:
>
> In database/ioc/src/Makefile 
> <https://github.com/epics-base/pvExampleTestCPP/pull/1#discussion_r75035740>:
>
> > @@ -1,4 +1,4 @@
> > -TOP=../..
> > +TOP=../../..
>
> There should be an entry in the |database/configure/RELEASE| file 
> pointing to |$(TOP)/..|, which is how this is handled in the other 
> embedded tops in this module. My guess is that when he worked on this 
> code, Marty's development workspace had a |RELEASE.local| file in the 
> parent V4 directory that included a pointer to the pvExampleTestCPP 
> directory, since that would also provide a way for the build to find 
> that powerSupply stuff from |pvExampleTestCPP/src| and would explain 
> why he never saw the problem.
>
> Given that this module has been superceded by moving its code into the 
> enhanced exampleCPP module we don't really want to spend much time on 
> this, and I've suggested that Marty remove everything except an 
> explanatory README.md file from the master branch to document that. 
> I'd be happy to merge a modified version with your other changes into 
> the release/4.1 branch for users still on the V4.5.0.x series if you 
> think that's worth doing.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/epics-base/pvExampleTestCPP/pull/1/files/5b841497a18bbed3e022c263f52aee840ed4475f#r75035740>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AF1Q1vCQf9YGf3-Ch6ear_nGBwkwQcsGks5qgj8QgaJpZM4JlC19>.
>


Maybe instead of removing everything except the README.md just change 
the README.md so that it clearly states that starting with V4.6 all 
examples have been moved to exampleCPP?

Note that it appears that the pull request from Bruce is to the master 
branch.